react/README.md

```diff
@@ -40,3 +40,22 @@
 [keys]: https://reactjs.org/docs/lists-and-keys.html#keys
 [component composition over component inheritance]: https://reactjs.org/docs/composition-vs-inheritance.html
 [react router]: https://reacttraining.com/react-router/
+
+## General Philosophies
+
+- For greenfield React projects we like to use TypeScript. TypeScript is a typed superset of JavaScript that compiles to
+  plain JavaScript. For a quick introduction, check out
+  [TypeScript in 5 minutes](https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes.html).
+- If our app needs to make network requests and hold state outside of a component, we like using GraphQL and our
+  preferred library is [Apollo](https://www.apollographql.com). We have a [section on `GraphQL`](/graphql/) in our
+  Guides.
+- When building React apps with TypeScript and Apollo, we've found working in [VSCode](https://code.visualstudio.com/)
+  to be a mostly-good experience.
+
+## Create App
+
+Use [react-suspenders](https://github.com/thoughtbot/react-suspenders):
```

Comment:
This is still a private project. Did you mean to link it? Or maybe you meant to make react-suspenders public?

Reply:
It is still private, I should not link it.